rtsp.go

```diff
@@ -457,6 +457,10 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 
 		for _, nalu := range au {
 			frame, err := rc.rawDecoder.decode(nalu)
+			var recoverableErr *RecoverableAVError
+			if errors.As(err, &recoverableErr) {
+				continue
```

Comment:
Are there any differences between the go standard library's `errors.As` and the cursed one's? Cursed on is imported in this file.

Reply:
Nah

Will remove the cursed pkg in https://viam.atlassian.net/browse/RSDK-8776

Can't do it as a quick one-liner since we have to refactor errors.wrap which is exclusive to the cursed pkg